js/modules/k6/crypto/crypto.go

```diff
@@ -68,114 +115,157 @@ func (*Crypto) RandomBytes(ctx context.Context, size int) *goja.ArrayBuffer {
 	return &ab
 }
 
-// Md4 returns the MD4 hash of input in the given encoding.
-func (c *Crypto) Md4(ctx context.Context, input interface{}, outputEncoding string) interface{} {
-	hasher := c.CreateHash(ctx, "md4")
+// MD4 returns the MD4 hash of input in the given encoding.
+func (c *Crypto) MD4(input interface{}, outputEncoding string) interface{} {
```

Comment:
You could just as well make them unexported at this point. I am not particularly certain they should be used by some other go module :shrug: